codecov.sh

```diff
@@ -0,0 +1,1696 @@
+#!/usr/bin/env bash
```

Comment:
Is this the only way to get this script? Can't pull it in as part of a dependency?